.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,21 @@
+---
+name: "Documentation"
+about: Report an issue related to the NumPy documentation
+
+---
+
+## Documentation
+
+<!-- If this is an issue with the current documentation for NumPy (i.e.
+incomplete/innacurate docstring, unclear explanation in any part of the
+documentation), make sure to leave a reference to the document/code you're
+referring to. -->
+
+<!-- If this is an idea or a request for content, please describe as clearly as
+possible what topics you think are missing from the current documentation. Make
+sure to check https://github.com/numpy/numpy-tutorials and see if this issue
+might be more appropriate there. -->
+
+<!-- If this is about the content in https://numpy.org, please open an issue at
```

Comment:
This is going to confuse users looking at a url like `https://numpy.org/devdocs/reference/generated/numpy.ndindex.html`

Reply:
I think we should leave this one out for now. It would be easy enough to mark such issues with a label and/or move them to that repo.

Reply:
I agree. Do you think there's a way to make it clearer to differentiate between those sources, or should I just remove this line entirely?